src/styles/_search-bar.scss

```diff
@@ -0,0 +1,45 @@
+#search-bar-div {
+  position: absolute;
+  left: 33%;
+  top: 15px;
+  width: 65%;
```

Comment:
The width of the search bar looks a little too wide to me on Desktop:

![image](https://user-images.githubusercontent.com/161748/83872604-29f80e00-a6e7-11ea-97a7-4a43cd50127a.png)

Setting it to `35rem` - `40rem` (350px - 400px) seems to be sufficient?

![image](https://user-images.githubusercontent.com/161748/83872789-865b2d80-a6e7-11ea-9ae6-580585917bff.png)

Reply:
Made it smaller